test/unit/messaging/schemas/remaining-amount.test.js

```diff
@@ -1,7 +1,6 @@
-const REMAINING_AMOUNT = require('../../../mocks/components/remaining-amount')
-
+const { topUpRemainingAmount } = require('../../../mocks/components/remaining-amount')
+const REMAINING_AMOUNT = topUpRemainingAmount
 const schema = require('../../../../app/messaging/schemas/remaining-amount')
-
```

Comment:
I think the whitespace aids readability between imports and variable value that will change between test cases